spec.md

```diff
@@ -1490,7 +1490,6 @@ If `pull_policy` and `build` both presents, Compose implementations SHOULD build
 ```
 
 ### runtime
-_DEPRECATED: this attribute is low-level platform implementation detail_ 
 
 `runtime` specifies which runtime to use for the service’s containers.
```

Comment:
if we embrace this attribute, should then be correctly documented.
Can you at least include a link to https://github.com/opencontainers/runtime-spec/blob/master/implementations.md ?

Reply:
updated, sorry for the delay